src/mount_efs/__init__.py

```diff
@@ -241,10 +241,14 @@
 FEDORA_RELEASE_NAME = 'Fedora release'
 OPEN_SUSE_LEAP_RELEASE_NAME = 'openSUSE Leap'
 SUSE_RELEASE_NAME = 'SUSE Linux Enterprise Server'
+ALMALINUX_8_4_RELEASE_NAME = 'AlmaLinux release 8.4 (Electric Cheetah)'
```

Comment:
Is that possible we just do `AlmaLinux release 8` here? So that we can support AlmaLinux 8.x, not only 8.4

Reply:
I tried doing that and back then it wasn't working. I can add a separate line for supporting 8.x.